helm/influxdb-athena-crawler/README.md

```diff
@@ -56,15 +56,18 @@ helm install influxdb-athena-crawler influxdb-athena-crawler/influxdb-athena-cra
 | defaults.region | string | `""` | The AWS region. |
 | defaults.bucket | string | `""` | The AWS bucket to watch. |
 | defaults.prefix | string | `""` | The bucket prefix. |
+| defaults.suffix | string | `""` | The bucket files suffix. |
+| defaults.processedFlagSuffix | string | `""` | The bucket processed flags suffix. |
 | defaults.cleanObjects | bool | `false` | Whether to delete S3 objects after processing them. |
+| defaults.maxObjectAge | string | `"5m"` | After how long to delete the objects. |
 | defaults.timeout | string | `"10m"` | The global timeout. |
 | defaults.influxServers | list | `[]` | The InfluxDB servers addresses. |
 | defaults.influxToken | string | `""` | The InfluxDB token. |
 | defaults.influxOrg | string | `""` | The InfluxDB org to write to. |
 | defaults.influxBucket | string | `""` | The InfluxDB bucket write to. |
 | defaults.measurement | string | `""` | The InfluxDB bucket measurement. |
 | defaults.timestampRow | string | `"timestamp"` | The timestamp row in CSV. |
-| defaults.timestampLayout | string | `"2006-01-02T15:04:05.000Z"` | The layout to parse timestamp. |
```

Comment:
Is that related ?

Reply:
Good catch, it is intended yes, looks like I failed to regenerate the doc during the work on #332 
The idea is to have the default value match the format usually present in the CSVs.

Reply:
Ok !